bundles/org.openhab.binding.bmwconnecteddrive/.gitignore

```diff
@@ -0,0 +1 @@
+/.README.md.html
```

Comment:
Did you add this by intention?

Reply:
Yes, ignoring the precompiled readme.md from eclipse. Is this a problem?

Reply:
It would be a bit strange to only ignore it in your binding. You can define ignore patterns in your local git configuration if the file bothers you.

Reply:
ok